app/views/stafftools/impersonate.js.erb

```diff
@@ -0,0 +1 @@
+$('#js-user-search-results').html("<%= escape_javascript(render partial: 'stafftools/impersonate/user_results', locals: { users: @users }) %>");
```

Comment:
You can have an endpoint that renders the partial without a layout. In JavaScript you can GET that endpoint and update the DOM.